src/client.ts

```diff
@@ -6107,11 +6163,11 @@ export class MatrixClient extends EventEmitter {
     public register(
         username: string,
         password: string,
-        sessionId: string,
+        sessionId: string | null,
```

Comment:
Note for myself, these type changes should be split out into another PR. I need them for matrix-video-call.

Reply:
Split out into https://github.com/matrix-org/matrix-js-sdk/pull/1903